jenkins_pipelines/environments/manager-4.2-qe-build-validation

```diff
@@ -35,7 +35,7 @@ node('sumaform-cucumber-provo') {
             choice(name: 'rake_namespace', choices: ['parallel', 'cucumber'], description: 'Choose parallel or cucumber'),
             string(name: 'capybara_timeout', defaultValue: '30', description: 'Capybara max. waiting time'),
             string(name: 'default_timeout', defaultValue: '300', description: 'Default timeout used in our Test Framework'),
-            text(name: 'custom_repositories', defaultValue: '{"server":{"server_stack":"","salt15sp2_base":"","salt15sp2_python2_module":"","salt15sp2_server_apps_module":"","extra_sle_that_affects_suma_srv":""},"proxy":{"salt":"","tools":"","salt15sp2":"","extra_sle_that_affects_suma_proxy":""},"sle12sp4_minion":{"salt":"","tools":""},"sle12sp4_client":{"tools":""},"sle12sp5_minion":{"salt":"","tools":""},"sle12sp5_client":{"tools":""},"sle15_minion":{"salt":"","tools":""},"sle15_client":{"tools":""},"sle15sp1_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp1_client":{"tools":""},"sle15sp2_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp2_client":{"tools":""},"sle15sp3_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp3_client":{"tools":""},"debian9_minion":{"salt":"","tools":""},"debian10_minion":{"salt":"","tools":""},"ubuntu1804_minion":{"salt":"","tools":""},"ubuntu2004_minion":{"salt":"","tools":""},"ceos7_minion":{"salt":"","tools":""},"ceos7_client":{"tools":""},"rocky8_minion":{"salt":"","tools":""},"rocky8_client":{"tools":""},"opensuse154arm_minion":{"salt":"","tools":""}}', description: 'Salt & Client Tools MU Repositories for each client, in json format')
+            text(name: 'custom_repositories', defaultValue: '{"server":{"server_stack":"","salt15sp2_base":"","salt15sp2_python2_module":"","salt15sp2_server_apps_module":"","extra_sle_that_affects_suma_srv":""},"proxy":{"salt":"","tools":"","salt15sp2":"","extra_sle_that_affects_suma_proxy":""},"sle12sp4_minion":{"salt":"","tools":""},"sle12sp4_client":{"tools":""},"sle12sp5_minion":{"salt":"","tools":""},"sle12sp5_client":{"tools":""},"sle15_minion":{"salt":"","tools":""},"sle15_client":{"tools":""},"sle15sp1_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp1_client":{"tools":""},"sle15sp2_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp2_client":{"tools":""},"sle15sp3_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp3_client":{"tools":""},"debian10_minion":{"salt":"","tools":""},"ubuntu1804_minion":{"salt":"","tools":""},"ubuntu2004_minion":{"salt":"","tools":""},"ceos7_minion":{"salt":"","tools":""},"ceos7_client":{"tools":""},"rocky8_minion":{"salt":"","tools":""},"rocky8_client":{"tools":""},"opensuse154arm_minion":{"salt":"","tools":""}}', description: 'Salt & Client Tools MU Repositories for each client, in json format')
```

Comment:
So, CentOS 7 still supported? (I'm not sure)

Reply:
it is

Reply:
Yes. See https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.3/index.html#_unsupported_products

Reply:
And similar file for 4.2.